src/uikit.cs

```diff
@@ -13970,6 +14520,34 @@ interface UIView : UIAppearance, UIAppearanceContainer, UIAccessibility, UIDynam
 		[TV (13,0), NoWatch, iOS (13,0)]
 		[Export ("overrideUserInterfaceStyle", ArgumentSemantic.Assign)]
 		UIUserInterfaceStyle OverrideUserInterfaceStyle { get; set; }
+
+		[NoWatch, TV (12,0), iOS (12,0)] // Yes, 12 and intro is happy on 12 device.
+		[Export ("transform3D", ArgumentSemantic.Assign)]
+		CATransform3D Transform3D { get; set; }
+
+#if !TVOS // avoid a CS0109 also these are not in TVOS anyways...
```

Comment:
This might be more self-explanatory:

```csharp
#if TVOS
#pragma warning disable 0109 // The member 'member' does not hide an inherited member. The new keyword is not required
#endif
...
#if TVOS
#pragma warning restore 0109
#endif